libs/utils/platforms/juno_energy.py

```diff
@@ -0,0 +1,109 @@
+from energy_model import ActiveState, EnergyModelNode, PowerDomain, EnergyModel
+
+from collections import OrderedDict
+
+a53_cluster_active_states = OrderedDict([
+    (450000, ActiveState(power=26)),
+    (575000, ActiveState(power=30)),
+    (700000, ActiveState(power=39)),
+    (775000, ActiveState(power=47)),
+    (850000, ActiveState(power=57)),
+])
+
+# TODO warn if any of the idle states aren't represented by power domains
+a53_cluster_idle_states = OrderedDict([
+    ("WFI",               56),
+    ("cpu-sleep-0",       56),
+    ("cluster-sleep-0",   17),
+])
+
+a53_cpu_active_states = OrderedDict([
+    (450000, ActiveState(capacity=235, power=33)),
+    (575000, ActiveState(capacity=302, power=46)),
+    (700000, ActiveState(capacity=368, power=61)),
+    (775000, ActiveState(capacity=406, power=76)),
+    (850000, ActiveState(capacity=447, power=93)),
+])
+
+a53_cpu_idle_states = OrderedDict([
+    ("WFI",               6),
+    ("cpu-sleep-0",       0),
+    ("cluster-sleep-0",   0),
+])
+
+a53s = [0, 3, 4, 5]
```

Comment:
Hmm, that's a good point. I'd really like for this class to be usable without a connected target, so I'm not sure what the solution is here. Probably worth discussing this F2F